src/utils/common-utils.ts

```diff
@@ -11,184 +11,49 @@ export enum ResourceType {
  * https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/initiatorType
  */
 export enum InitiatorType {
-    /**
-     * IMAGES
-     * PerformanceResourceTiming with initiatorType=Input must be an image
-     * Per MDN docs: "if the request was initiated by an <input> element of type image.""
-     */
+    // IMAGES
```

Comment:
question: there are other valid initiatorTypes listed in the doc. Are these not needed?

Reply:
There are many other initiatorTypes, but they do not align with our RUM-specific choices for fileType. There may be something that I missed since there are so many initatorTypes, but from my deep dive, these choices cover what we need. 

I also left another comment that talks about this choice. Was also planning on updating documentation when all the discussions are done. https://github.com/aws-observability/aws-rum-web/pull/489#discussion_r1448109111